modules/140-user-authz/templates/cluster-roles.yaml

```diff
@@ -214,6 +214,12 @@
   verbs:
   - delete
   - deletecollection
+- apiGroups:
```

Comment:
There is an example in the issue — https://github.com/deckhouse/deckhouse/blob/main/modules/150-user-authn/templates/user-authz-cluster-roles.yaml. I didn't mention it, but you just need to create ClusterRole and add the annotation to it:
```
    user-authz.deckhouse.io/access-level: Editor
```